### PR TITLE
adding logo link + svg

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
 <link rel=stylesheet href=/static/css/edit.css>
 <script src=/static/js/utils.js></script>
 
-<h1>Cafebabel</h1>
+<a href=# class=logo>
```
Comment (vinyll) [inline templates/base.html]: `href={{ url_for('home') }}` will insert the url for the homepage (probably `/`).

**Review by vinyll — CHANGES_REQUESTED** [review]

File: templates/base.html (hunk 1 above)
Comment (johangiraud) [reply templates/base.html]: yes !

File: templates/base.html [hunk 2]
```diff
@@ -7,7 +7,9 @@
 <link rel=stylesheet href=/static/css/edit.css>
 <script src=/static/js/utils.js></script>
 
-<h1>Cafebabel</h1>
+<a href={{ url_for('home') }} class=logo>
+  <img src=https://rawgit.com/cafebabel/styleguide/master/img/logo-cafebabel.svg alt="logo cafébabel">
```
Comment (vinyll) [inline templates/base.html]: Eventually _logo Cafébabel_ with a capital C, or not?

**Review by vinyll — APPROVED** [review]

File: templates/base.html (hunk 2 above)
Comment (johangiraud) [reply templates/base.html]: oui ! mieux

**Pull request merged**